app/controllers/new_admin/users_controller.rb

```diff
@@ -7,6 +7,7 @@ class UsersController < ApplicationController
     def show
       @user = User.find(params[:id]).decorate
       @user_allocations = Allocation.where(user_id: params[:id]).decorate
+      @performance_evaluations = Evaluation.where(evaluated_id: params[:id])
```

Comment:
Won't just querying the evaluations table retrieve the english evaluations too?

Reply:
Yes! I just noticed this when I started in the english evaluations page, this will be fixed :+1: